docs/tutorials/configure/pet-photographer.md

```diff
@@ -0,0 +1,984 @@
+For example, the [sensors service](/services/sensors/) provides a central interface to all of your robot’s sensors.
+
+### Add data management service
+
+To enable data capture on your robot, add and configure the [data management service](/services/data/) to capture and store data on your robot's computer:
+
+{{< tabs >}}
+{{% tab name="Builder" %}}
+
+1. On the **Config** page, click the **Services** subtab and click **Create service** in the lower-left corner.
+1. Choose `Data Management` as the type and name your instance of the data manager `dm`.
+   This service syncs data from your robot to the Viam app in the cloud.
+1. Select **Create**.
+1. On the panel that appears, you can manage the capturing and syncing functions individually.
+   By default, the data management service captures data every 0.1 minutes in the <file>~/.viam/capture</file> directory.
+
+   You can leave the default settings as they are.
+   Click **Save config** at the bottom of the window.
+
+   ![An instance of the data management service named "dm". The cloud sync and capturing options are toggled on and the directory is empty. The interval is set to 0.1](/tutorials/pet-photographer/data-management-services.png)
+
+   For more detailed information, see [Add the data management service](/services/data/configure-data-capture/#add-the-data-management-service).
+   {{% /tab %}}
+   {{% tab name="JSON Template" %}}
+   Add the vision service object to the services array in your rover’s raw JSON configuration:
+
+```json {class="line-numbers linkable-line-numbers"}
+    {
+      "name": "dm",
+      "type": "data_manager",
+      "namespace": "rdk",
+      "attributes": {
+        "sync_interval_mins": 0.1,
+        "capture_dir": "",
+        "tags": [],
+        "additional_sync_paths": []
+      }
+    },
+  ... // Vision and other services
+```
+
+Click **Save Config** and head back to **Builder** mode.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Vision service to detect color
+
+This tutorial uses the color of my dog's collar, `#43A1D0` or `rgb(67, 161, 208)` (blue), but you can use a different color that matches your pet or a distinctly colored item on your pet.
+
+**Hex color #43A1D0**: {{<imgproc src="/tutorials/pet-photographer/43a1d0.png" resize="90x" declaredimensions=true alt="A color swatch for the color of the example subject's collar">}}
+
+To configure your [vision service color detector](/services/vision/detection/):
+
+{{< tabs >}}
+{{% tab name="Builder" %}}
+
+1. At the bottom-left of the page, click **Create service**.
+1. Select the `Vision` type, then select the `Color Detector` model.
+1. Enter `my_color_detector` as the name for your detector and click **Create**.
+1. In the vision service panel, click the color selection box to set the color to be detected.
+   For this tutorial, set the color to the color of your pet, or use a blue collar or ribbon to improve your filter's accuracy.
```

Comment:
Nit: try to keep procedural portions as actionable as possible. For this tutorial, enter this hex code.

```suggestion
1. In the vision service panel, click the color selection box to set the color to be detected.
   For this tutorial, set the color to `#43A1D0` or `rgb(67, 161, 208)`.
   Alternatively, you can provide the color of your pet, or use a different brightly-colored collar or ribbon.
```